### PR TITLE
Logo change

### PR DESCRIPTION
**Pull request by ksol**

### CODE REVIEW

**Review by yannski — CHANGES_REQUESTED** [review]
In the footer, can you:
- remove the "Add-ons" and "Scalingo Enterprise" entries
- remove the "Media Kit & Logos" entry 
- remove the "Newsletters" entry
- rename the "Technologies" into "Runtimes"

File: _includes/footer.html [hunk 1]
```diff
@@ -1,7 +1,49 @@
 <footer class="realfooter d-print-none" id="footer"><div class="container">
 <div class="row mb-5">
 <div class="col-12 col-md-3 col-lg-1-5">
-<svg class="scalingo-logo-inverse" viewbox="0 0 136.35483 35.9" xmlns="http://www.w3.org/2000/svg"><use xlink:href="#scalingo-logo-full-white"></use></svg></div>
+  <svg viewBox="0 0 111 28" style="height: 36px; width: initial;" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (EtienneM) [inline _includes/footer.html:4]: Do you want to add a comment what this SVG is, where does it come from? To ease later read of this code

File: package.json [hunk 2]
```diff
@@ -18,5 +18,9 @@
   },
   "devDependencies": {
     "webpack-dev-server": "^2.11.1"
+  },
+  "volta": {
```
Comment (EtienneM) [inline package.json:22]: Out of curiosity, what is this `volta` section about?

**Review by EtienneM — APPROVED** [review]
You may want to remove the file `favicon-32x32.png` as you include the favicon from DatoCMS

File: package.json (hunk 2 above)
Comment (ksol) [reply package.json:22]: it's js tooling, volta.sh

**Review by yannski — APPROVED** [review]

**Pull request merged**